pyproject.toml

```diff
@@ -15,7 +15,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-click = "^7.1"
+click = "^7.1 || ^8.0"
```

Comment:
Interesting, I didn't know this was a supported syntax! I would have expected to see `">= 7.1, < 9.0"`, but if this works too, fine by me. :-)